bounswe/bounswe2020group7
# Add: Ratings Feature

Issue: [Comment Ratings](https://github.com/bounswe/bounswe2020group7/issues/429)

## Summary
Added comments & rating feature as a rating modal and profile page widget.

Looking forward to add comments tab to profile page. I can complete this PR after [#435](https://github.com/bounswe/bounswe2020group7/issues/435) is solved.

## QA Test Plan
- Go to profile page of another user that you have a common workspace
- Click rate button
**Expected:** Rating dialogue should be open
- Choose a rating and type a comment
- Click submit
**Expected:** Success notification appears. Popup is closed.
